modules/pool/lib/apr-data-sources/base-apr-handlers/sources/default-apr-handler.ts

```diff
@@ -0,0 +1,62 @@
+import axios from 'axios';
+
+import { AprHandler } from '../base-apr-handlers';
+
+export class DefaultAprHandler implements AprHandler {
+    tokens: {
+        [tokenName: string]: string;
+    };
+    url: string;
+    path: string;
+    scale: number;
+    networkPrismaId: string;
+    group = 'DEFAULT';
+
+    constructor(aprHandlerConfig: DefaultAprHandlerConfig) {
+        this.tokens = aprHandlerConfig.tokens;
+        this.url = aprHandlerConfig.sourceUrl;
+        this.networkPrismaId = aprHandlerConfig.networkPrismaId;
+        this.path = aprHandlerConfig.path ?? '';
+        this.scale = aprHandlerConfig.scale ?? 100;
+        this.group = aprHandlerConfig.group ?? 'DEFAULT';
```

Comment:
this should be empty or undefined and only set if needed, otherwise if a pool as reth and wsteth apr it would be grouped under DEFAULT in the UI.